extensions/indexes/range/src/org/exist/indexing/range/ComplexRangeIndexConfigElement.java

```diff
@@ -162,9 +163,9 @@ public boolean matchConditions(Node node) {
         return true;
     }
 
-    public boolean findCondition(QName lhe, String rhe, RangeIndex.Operator operator) {
+    public boolean findCondition(Predicate predicate) {
```

Comment:
In future please consider using `final` on all variables that are not reassigned.